src/emitters/emitter.sqns.ts

```diff
@@ -676,6 +719,7 @@ export class SqnsEmitter implements IEmitter {
     const results = await Promise.allSettled(
       messages.map((message) => this.processMessage(message))
     );
+    // Will this not delete the message even when it failed, don't know for sure - please check
```

Comment:
The deleteMessage function handles the successful and failed messages

Reply:
This can be removed

Reply:
Okies

Reply:
Done.